.github/ISSUE_TEMPLATE/a11y_report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Accessibility Report
+about: Tell us about something that is not accessible.
+title: ''
+labels: a11y
+assignees: ''
+---
+## Missing Alt-text
+Please do not currently report missing alt-text for images unless you are part of the alt-text audit.
```

Comment:
I would remove this for now. It would be good to have all known instances of alt-text missing from images logged in GitHub and users can use the search function to see if theirs has been reported already

Reply:
This is only intended to be there a few weeks while I make an initial list of the missing alt-text.  It's to save *me* from having to double check every line of that list against any reports so that I can have a more efficient workflow while creating that first list.  After that, I absolutely agree this should be removed.